include/accessor.h

```diff
@@ -522,6 +540,13 @@ class accessor<DataT, Dims, Mode, target::host_task> : public detail::accessor_b
 	// m_host_ptr must be defined *last* for it to overlap with the sequence of range and id members in the 0-dimensional case
 	CELERITY_DETAIL_NO_UNIQUE_ADDRESS DataT* m_host_ptr = nullptr;
 
+#if CELERITY_ACCESSOR_BOUNDARY_CHECK
+	id<3>* m_oob_indices = nullptr;
+	inline static std::mutex m_mutex;
```

Comment:
I'd call this `m_oob_mutex`, and maybe add a comment briefly documenting why it is `inline static`.